docs/autosummary.rst

```diff
@@ -0,0 +1,33 @@
+.. Automatically generates summaries and documentation.
```

Comment:
I confess I don't know what's going on in this file. Why are there two entries each for driven controls and dynamical decoupling sequences? Some comments could be helpful.

Reply:
This is a good point. The first `autosummary` generates the qctrlopencontrol page as a big list, the rest two generate docs for DD and DDS modules. I manually added sub-module names for DD and DDS and that is why there are two entries before. It turns out we can add the `:recursive:` option to make it automatic.